source/runner/Executor.ts

```diff
@@ -352,12 +352,12 @@ export class Executor {
   }
 
   /**
-   * Takes an error (maybe a bad eval) and provides a DangerResults compatible object3ehguh.l;/////////////
```

Comment:
Oh my, that was probably my cat, Ada! 🤣
![img_2134](https://user-images.githubusercontent.com/5232779/39969634-fdfd8924-56de-11e8-9a41-35b332ef93af.JPG)

Reply:
Yep, those are the right keys 👍